tasks/send-slack-notification/README.md

```diff
@@ -0,0 +1,11 @@
+# send-slack-notification task
+
+Sends message to Slack using postMessage API
+
+## Parameters
+| Name            | Description                                                | Optional | Default Value             |
+|-----------------|------------------------------------------------------------|----------|---------------------------|
+| message         | Message to be sent                                         | false    |                           |
```

Comment:
The convention in this repo is to say "Yes" or "No" in the "Optional" column. So we should probably stick with that.

Reply:
Fixed